modules/core/core-api/src/main/java/com/enonic/xp/schema/content/ContentType.java

```diff
@@ -271,6 +271,15 @@ public Builder addFormItem( final FormItem formItem )
             return this;
         }
 
+        public Builder addFormItems( final List<FormItem> formItems )
```

Comment:
ContentType is not a collection of form items. It is a holder of a Form. I guess addFormItem method is wrong and should be removed, not reinforced